### PR TITLE
Add semaphore badge and remove coveralls badge

### PR DESCRIPTION
**Pull request by tomdee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [![CircleCI branch](https://img.shields.io/circleci/project/projectcalico/calico-cni/master.svg)](https://circleci.com/gh/projectcalico/calico-cni/tree/master)
-[![Coverage Status](https://coveralls.io/repos/projectcalico/calico-cni/badge.svg?branch=master&service=github)](https://coveralls.io/github/projectcalico/calico-cni?branch=master)
+[![Build Status](https://semaphoreci.com/api/v1/calico/calico-cni-2/branches/master/badge.svg)](https://semaphoreci.com/calico/calico-cni-2)
```
Comment (caseydavenport) [inline README.md]: Why does it need to be calico-cni-2?

Comment (tomdee) [inline README.md]: It's an autogenerated name. I have no control over it...

Comment (caseydavenport) [inline README.md]: Well, keep auto-generating it until it says `calico-cni`.

**Comment by caseydavenport** [comment]
LGTM.

**Pull request merged**